CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 All notable changes to this project will be documented in this file.
 
+## [4.0.0](https://github.com/voxpupuli/voxpupuli-puppet-lint-plugins/tree/4.0.0) (2023-02-28)
+
+[Full Changelog](https://github.com/voxpupuli/voxpupuli-puppet-lint-plugins/compare/3.1.0...4.0.0)
+
+**Breaking changes:**
+
+- Drop Ruby 2.4 support  [\#36](https://github.com/voxpupuli/voxpupuli-puppet-lint-plugins/pull/36) ([bastelfreak](https://github.com/bastelfreak))
+- Require puppet-lint 3.1 [\#33](https://github.com/voxpupuli/voxpupuli-puppet-lint-plugins/pull/33) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
Should this mention 3.2 too?

Reply:
I updated it to `require puppet-lint 3.1 or newer`